### ISSUE TITLE
When re-login after switching user, takes to the lock-screen

### ISSUE THREAD

**Issue by bsucker98** [budgie-desktop/budgie-desktop#1388]
When reporting issues for Budgie Desktop, ensure you read these guidelines and delete this first
introductory paragraph. If including build failures, or general tool output, please ensure they are
in English, i.e. `LC_ALL=C LANG=C`. If your issue is a Budgie *crash*, include a stack trace. (gdb)

**Operating System**: e.g. Solus 1.2.0.5
**Budgie version**: i.e. Budgie 10.2.7 OR specific git commit

Before submitting *ANY* issue, please run the following command to
verify that you have, at minimum, Budgie `10.2.7`, to provide the
exact version of your Budgie installation:

    budgie-desktop --version

**The issue encountered**

srry, if it's expected to be that way,
when re-login(already logged) after switching user, it takes the user to lock-screen & has to re-type the password.

**Steps to reproduce the issue**

should take to the desktop directly when re-login, through login screen, to an already logged account instead of taking to lock-screen.

**Comment by fossfreedom** [comment]
doesn't sound like a budgie-desktop issue per se - but likely a gnome-screensaver / lightdm handoff interaction issue.

Just to clarify are you on the lock screen when you click "switch user" ?

**Comment by bsucker98** [comment]
>@fossfreedom are you on the lock screen when you click "switch user" ?

yes. I'm on the lock screen, given that u cannot switch user directly from the desktop(may be i don't know), without locking screen..& clicking "switch user" takes to login screen
